Cargo.toml

```
@@ -68,6 +68,7 @@ serde = { version = "1.0.193", default-features = false }
 serde_json = "1.0.108"
 serde_with = { version = "3.4.0", default-features = false }
 sha2 = { version = "0.10.8", default-features = false }
+strum = "0.26.2"
```

```suggestion
strum = { version="0.26.2", default-features = false }
```
The only place where default features can be disabled in a workspace dep in a workspace cargo project is here. Anywhere else is ignored by cargo.

So the only way to be of the feature-minimal mindset is to disable default features here and only enable them where needed. (even though it might only have this one usage rn).

---

Thank you for the comment, I will remember it for the future. I removed the dep completely as per above though. :)